x-pack/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -65,6 +65,7 @@ export class HeadlessChromiumDriverFactory {
       logger.info(`Creating browser page driver`);
 
       const chromiumArgs = this.getChromiumArgs(viewport);
+      logger.debug(`Chromium launch args set to: ${chromiumArgs}`);
```

Comment:
This log shows that we are setting `--window-size` in the args, although I don't think the requested window size actually is getting set on launch.